ansible/playbook/tasks/update.yml

```diff
@@ -17,17 +17,28 @@
     - name: Check reread results
       debug: var=reread_result.stdout_lines
 
-    # split download and update to produce a bit more of progress output
-    - name: Download packages
+    - name: Update pmm2 packages
       yum:
-        name: '*'
+        name:
+          - percona-grafana
+          - percona-prometheus
+          - percona-qan-api2
+          - percona-qan-app
+          - percona-dashboards
+          - pmm-server
+          - pmm-managed
+          - pmm-update
+          - percona-clickhouse-client
+          - percona-clickhouse-server
+          - pmm2-client
         state: latest
-        download_only: yes
-    - name: Update packages
+    
+    - name: Update system packages
       yum:
         name: '*'
         state: latest
-
+        security: yes
+        disablerepo: "percona-release-x86_64,percona-release-noarch,pmm2-server"
```

Comment:
We don't need that line there. We discussed it as a part of alternative solution.